sdcm/utils/benchmarks.py

```diff
@@ -42,6 +84,7 @@ def install_benchmark_tools(self):
             runner.install_benchmark_tools()
 
     def run_benchmarks(self):
+        # ParallelObject(self._benchmark_runners).run(lambda x: x.run_benchmarks())
```

Comment:
?

Reply:
Fixed with rebase (the commented out code was not present in the latter PR commits, now it won't show up in the earlier ones either)